config.yaml

```diff
@@ -18,15 +18,24 @@ timeline:
 # Cameras and other input data sources
 # Using Home Assistant conventions to ease upcoming integration
 sources:
-#  front_door_camera:
-#    uri: *secret_uri_front_door_camera
-    # type: video, audio, or auto
-#    type: video
-#    live: true
-#  entry_area_camera:
-#    uri: *secret_uri_entry_area_camera
-#    type: video
-#    live: true
+
+  # front_door_camera:
+  #   uri: *secret_uri_front_door_camera
+  #   type: video, audio, or auto
+  #   type: video
+  #   live: true
+
+  # local video device integration example
+  webcam:
+    uri: /dev/video0
```

Comment:
Which type of device and OS is this tested to work on? Any Linux, Mac, Win?
Can you please add a few lines to the [config docs](https://docs.ambianic.ai/users/configure/) to set user expectations.

Reply:
You are right, focusing on linux (because of v4l2). It works with picamera also if the [kernel module is enabled](https://raspberrypi.stackexchange.com/a/26386/41277).

I do not have a windows / mac machine, I suspect won't work out of the box anyway